cfme/tests/integration/test_cfme_auth.py

```diff
@@ -11,7 +11,9 @@
 
 
 def pytest_generate_tests(metafunc):
-    auth_modes = cfme_data['auth_test_data']
+    auth_modes = cfme_data.get('auth_test_data')
+    if not auth_modes:
+        return
```

Comment:
should we for consistence parametrize with a empty list to trigger consistency in collection?